From 9d36ff90d3a488caa8cd7b3351e5a3bdc4229b1b Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 6 Mar 2020 21:04:05 +0000
Subject: [PATCH] init-env.in: do not add C/CXX parameters

These are taken from the cross environment and include
sysroot paths, so are not reproducible.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 gettext-tools/tests/init-env.in | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gettext-tools/tests/init-env.in b/gettext-tools/tests/init-env.in
index 0968388..3d9a8a5 100644
--- a/gettext-tools/tests/init-env.in
+++ b/gettext-tools/tests/init-env.in
@@ -3,10 +3,6 @@ top_builddir=../..
 
 OBJEXT="@OBJEXT@"
 EXEEXT="@EXEEXT@"
-CC="@CC@"
-CFLAGS="@CFLAGS@"
-CXX="@CXX@"
-CXXFLAGS="@CXXFLAGS@"
 OBJC="@OBJC@"
 OBJCFLAGS="@OBJCFLAGS@"
 CPPFLAGS="@CPPFLAGS@"
